All new errors:

### besluiten
```
 100:22  error  nlgov:paths-kebab-case  /besluit_verwerken is not kebab-case.  paths./besluit_verwerken
```

### catalogi
```
 1302:15  error  nlgov:query-keys-camel-case  domein__in is not lower camelCase.                paths./catalogussen.get.parameters[1].name
 1330:15  error  nlgov:query-keys-camel-case  rsin__in is not lower camelCase.                  paths./catalogussen.get.parameters[5].name
 2849:15  error  nlgov:query-keys-camel-case  omschrijving__icontains is not lower camelCase.   paths./informatieobjecttypen.get.parameters[3].name
 9079:15  error  nlgov:query-keys-camel-case  identificatie__icontains is not lower camelCase.  paths./zaaktypen.get.parameters[3].name
 9086:15  error  nlgov:query-keys-camel-case  omschrijving__icontains is not lower camelCase.   paths./zaaktypen.get.parameters[4].name
```

### documenten
```
  282:25  error  nlgov:paths-kebab-case       /document_registreren is not kebab-case.                      paths./document_registreren
  434:30  error  nlgov:paths-kebab-case       /documentnummer_reserveren is not kebab-case.                 paths./documentnummer_reserveren
  628:15  error  nlgov:query-keys-camel-case  creatiedatum__gte is not lower camelCase.                     paths./enkelvoudiginformatieobjecten.get.parameters[3].name
  635:15  error  nlgov:query-keys-camel-case  creatiedatum__lte is not lower camelCase.                     paths./enkelvoudiginformatieobjecten.get.parameters[4].name
  672:15  error  nlgov:query-keys-camel-case  objectinformatieobjecten__object is not lower camelCase.      paths./enkelvoudiginformatieobjecten.get.parameters[9].name
  678:15  error  nlgov:query-keys-camel-case  objectinformatieobjecten__objectType is not lower camelCase.  paths./enkelvoudiginformatieobjecten.get.parameters[10].name
  744:15  error  nlgov:query-keys-camel-case  trefwoorden__overlap is not lower camelCase.                  paths./enkelvoudiginformatieobjecten.get.parameters[16].name
 2573:15  error  nlgov:query-keys-camel-case  creatiedatum__gte is not lower camelCase.                     paths./enkelvoudiginformatieobjecten/_zoek.post.parameters[4].name
 2580:15  error  nlgov:query-keys-camel-case  creatiedatum__lte is not lower camelCase.                     paths./enkelvoudiginformatieobjecten/_zoek.post.parameters[5].name
 2617:15  error  nlgov:query-keys-camel-case  objectinformatieobjecten__object is not lower camelCase.      paths./enkelvoudiginformatieobjecten/_zoek.post.parameters[10].name
 2623:15  error  nlgov:query-keys-camel-case  objectinformatieobjecten__objectType is not lower camelCase.  paths./enkelvoudiginformatieobjecten/_zoek.post.parameters[11].name
 2689:15  error  nlgov:query-keys-camel-case  trefwoorden__overlap is not lower camelCase.                  paths./enkelvoudiginformatieobjecten/_zoek.post.parameters[17].name
 2972:15  error  nlgov:query-keys-camel-case  einddatum__gt is not lower camelCase.                         paths./gebruiksrechten.get.parameters[0].name
 2979:15  error  nlgov:query-keys-camel-case  einddatum__gte is not lower camelCase.                        paths./gebruiksrechten.get.parameters[1].name
 2986:15  error  nlgov:query-keys-camel-case  einddatum__lt is not lower camelCase.                         paths./gebruiksrechten.get.parameters[2].name
 2993:15  error  nlgov:query-keys-camel-case  einddatum__lte is not lower camelCase.                        paths./gebruiksrechten.get.parameters[3].name
 3018:15  error  nlgov:query-keys-camel-case  startdatum__gt is not lower camelCase.                        paths./gebruiksrechten.get.parameters[6].name
 3026:15  error  nlgov:query-keys-camel-case  startdatum__gte is not lower camelCase.                       paths./gebruiksrechten.get.parameters[7].name
 3034:15  error  nlgov:query-keys-camel-case  startdatum__lt is not lower camelCase.                        paths./gebruiksrechten.get.parameters[8].name
 3042:15  error  nlgov:query-keys-camel-case  startdatum__lte is not lower camelCase.                       paths./gebruiksrechten.get.parameters[9].name
```

### zaken

```
  567:25   error  nlgov:paths-kebab-case       /reserveer_zaaknummer is not kebab-case.                                                       paths./reserveer_zaaknummer
  1783:15  error  nlgov:query-keys-camel-case  betrokkeneIdentificatie__medewerker__identificatie is not lower camelCase.                     paths./rollen.get.parameters[1].name
  1790:15  error  nlgov:query-keys-camel-case  betrokkeneIdentificatie__natuurlijkPersoon__anpIdentificatie is not lower camelCase.           paths./rollen.get.parameters[2].name
  1796:15  error  nlgov:query-keys-camel-case  betrokkeneIdentificatie__natuurlijkPersoon__inpA_nummer is not lower camelCase.                paths./rollen.get.parameters[3].name
  1801:15  error  nlgov:query-keys-camel-case  betrokkeneIdentificatie__natuurlijkPersoon__inpBsn is not lower camelCase.                     paths./rollen.get.parameters[4].name
  1807:15  error  nlgov:query-keys-camel-case  betrokkeneIdentificatie__nietNatuurlijkPersoon__annIdentificatie is not lower camelCase.       paths./rollen.get.parameters[5].name
  1813:15  error  nlgov:query-keys-camel-case  betrokkeneIdentificatie__nietNatuurlijkPersoon__innNnpId is not lower camelCase.               paths./rollen.get.parameters[6].name
  1819:15  error  nlgov:query-keys-camel-case  betrokkeneIdentificatie__nietNatuurlijkPersoon__kvkNummer is not lower camelCase.              paths./rollen.get.parameters[7].name
  1825:15  error  nlgov:query-keys-camel-case  betrokkeneIdentificatie__nietNatuurlijkPersoon__vestigingsNummer is not lower camelCase.       paths./rollen.get.parameters[8].name
  1830:15  error  nlgov:query-keys-camel-case  betrokkeneIdentificatie__organisatorischeEenheid__identificatie is not lower camelCase.        paths./rollen.get.parameters[9].name
  1837:15  error  nlgov:query-keys-camel-case  betrokkeneIdentificatie__vestiging__kvkNummer is not lower camelCase.                          paths./rollen.get.parameters[10].name
  1843:15  error  nlgov:query-keys-camel-case  betrokkeneIdentificatie__vestiging__vestigingsNummer is not lower camelCase.                   paths./rollen.get.parameters[11].name
  1877:15  error  nlgov:query-keys-camel-case  machtiging__loa is not lower camelCase.                                                        paths./rollen.get.parameters[14].name
  3328:15  error  nlgov:query-keys-camel-case  tijdstip__gt is not lower camelCase.                                                           paths./substatussen.get.parameters[3].name
  3334:15  error  nlgov:query-keys-camel-case  tijdstip__gte is not lower camelCase.                                                          paths./substatussen.get.parameters[4].name
  3340:15  error  nlgov:query-keys-camel-case  tijdstip__lt is not lower camelCase.                                                           paths./substatussen.get.parameters[5].name
  3346:15  error  nlgov:query-keys-camel-case  tijdstip__lte is not lower camelCase.                                                          paths./substatussen.get.parameters[6].name
  3786:26  error  nlgov:paths-kebab-case       /zaak_afsluiten/{uuid} is not kebab-case.                                                      paths./zaak_afsluiten/{uuid}
  3842:26  error  nlgov:paths-kebab-case       /zaak_bijwerken/{uuid} is not kebab-case.                                                      paths./zaak_bijwerken/{uuid}
  3930:27  error  nlgov:paths-kebab-case       /zaak_opschorten/{uuid} is not kebab-case.                                                     paths./zaak_opschorten/{uuid}
  3985:21  error  nlgov:paths-kebab-case       /zaak_registreren is not kebab-case.                                                           paths./zaak_registreren
  4178:26  error  nlgov:paths-kebab-case       /zaak_verlengen/{uuid} is not kebab-case.                                                      paths./zaak_verlengen/{uuid}
  5864:15  error  nlgov:query-keys-camel-case  aangemaaktDoor__icontains is not lower camelCase.                                              paths./zaaknotities.get.parameters[1].name
  5875:15  error  nlgov:query-keys-camel-case  aanmaakdatum__gt is not lower camelCase.                                                       paths./zaaknotities.get.parameters[3].name
  5881:15  error  nlgov:query-keys-camel-case  aanmaakdatum__gte is not lower camelCase.                                                      paths./zaaknotities.get.parameters[4].name
  5887:15  error  nlgov:query-keys-camel-case  aanmaakdatum__lt is not lower camelCase.                                                       paths./zaaknotities.get.parameters[5].name
  5893:15  error  nlgov:query-keys-camel-case  aanmaakdatum__lte is not lower camelCase.                                                      paths./zaaknotities.get.parameters[6].name
  5919:15  error  nlgov:query-keys-camel-case  onderwerp__icontains is not lower camelCase.                                                   paths./zaaknotities.get.parameters[10].name
  5949:15  error  nlgov:query-keys-camel-case  tekst__icontains is not lower camelCase.                                                       paths./zaaknotities.get.parameters[15].name
  5960:15  error  nlgov:query-keys-camel-case  wijzigingsdatum__gt is not lower camelCase.                                                    paths./zaaknotities.get.parameters[17].name
  5966:15  error  nlgov:query-keys-camel-case  wijzigingsdatum__gte is not lower camelCase.                                                   paths./zaaknotities.get.parameters[18].name
  5972:15  error  nlgov:query-keys-camel-case  wijzigingsdatum__lt is not lower camelCase.                                                    paths./zaaknotities.get.parameters[19].name
  5978:15  error  nlgov:query-keys-camel-case  wijzigingsdatum__lte is not lower camelCase.                                                   paths./zaaknotities.get.parameters[20].name
  6865:26  error  nlgov:paths-kebab-case       /zaaknummer_reserveren is not kebab-case.                                                      paths./zaaknummer_reserveren
  8637:15  error  nlgov:query-keys-camel-case  archiefactiedatum__gt is not lower camelCase.                                                  paths./zaken.get.parameters[2].name
  8646:15  error  nlgov:query-keys-camel-case  archiefactiedatum__isnull is not lower camelCase.                                              paths./zaken.get.parameters[3].name
  8654:15  error  nlgov:query-keys-camel-case  archiefactiedatum__lt is not lower camelCase.                                                  paths./zaken.get.parameters[4].name
  8674:15  error  nlgov:query-keys-camel-case  archiefnominatie__in is not lower camelCase.                                                   paths./zaken.get.parameters[6].name
  8695:15  error  nlgov:query-keys-camel-case  archiefstatus__in is not lower camelCase.                                                      paths./zaken.get.parameters[8].name
  8711:15  error  nlgov:query-keys-camel-case  bronorganisatie__in is not lower camelCase.                                                    paths./zaken.get.parameters[10].name
  8733:15  error  nlgov:query-keys-camel-case  einddatumGepland__gt is not lower camelCase.                                                   paths./zaken.get.parameters[13].name
  8740:15  error  nlgov:query-keys-camel-case  einddatumGepland__lt is not lower camelCase.                                                   paths./zaken.get.parameters[14].name
  8747:15  error  nlgov:query-keys-camel-case  einddatum__gt is not lower camelCase.                                                          paths./zaken.get.parameters[15].name
  8753:15  error  nlgov:query-keys-camel-case  einddatum__isnull is not lower camelCase.                                                      paths./zaken.get.parameters[16].name
  8758:15  error  nlgov:query-keys-camel-case  einddatum__lt is not lower camelCase.                                                          paths./zaken.get.parameters[17].name
  8812:15  error  nlgov:query-keys-camel-case  identificatie__icontains is not lower camelCase.                                               paths./zaken.get.parameters[20].name
  8824:15  error  nlgov:query-keys-camel-case  kenmerk__bron is not lower camelCase.                                                          paths./zaken.get.parameters[22].name
  8897:15  error  nlgov:query-keys-camel-case  registratiedatum__gt is not lower camelCase.                                                   paths./zaken.get.parameters[29].name
  8905:15  error  nlgov:query-keys-camel-case  registratiedatum__lt is not lower camelCase.                                                   paths./zaken.get.parameters[30].name
  8913:15  error  nlgov:query-keys-camel-case  resultaat__resultaattype is not lower camelCase.                                               paths./zaken.get.parameters[31].name
  8919:15  error  nlgov:query-keys-camel-case  rol__betrokkene is not lower camelCase.                                                        paths./zaken.get.parameters[32].name
  8924:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__medewerker__identificatie is not lower camelCase.                paths./zaken.get.parameters[33].name
  8931:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__natuurlijkPersoon__anpIdentificatie is not lower camelCase.      paths./zaken.get.parameters[34].name
  8937:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__natuurlijkPersoon__inpA_nummer is not lower camelCase.           paths./zaken.get.parameters[35].name
  8942:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn is not lower camelCase.                paths./zaken.get.parameters[36].name
  8948:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__annIdentificatie is not lower camelCase.  paths./zaken.get.parameters[37].name
  8954:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__innNnpId is not lower camelCase.          paths./zaken.get.parameters[38].name
  8960:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__kvkNummer is not lower camelCase.         paths./zaken.get.parameters[39].name
  8966:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__vestigingsNummer is not lower camelCase.  paths./zaken.get.parameters[40].name
  8971:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__organisatorischeEenheid__identificatie is not lower camelCase.   paths./zaken.get.parameters[41].name
  8978:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__vestiging__kvkNummer is not lower camelCase.                     paths./zaken.get.parameters[42].name
  8984:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__vestiging__vestigingsNummer is not lower camelCase.              paths./zaken.get.parameters[43].name
  8990:15  error  nlgov:query-keys-camel-case  rol__betrokkeneType is not lower camelCase.                                                    paths./zaken.get.parameters[44].name
  9003:15  error  nlgov:query-keys-camel-case  rol__machtiging is not lower camelCase.                                                        paths./zaken.get.parameters[45].name
  9018:15  error  nlgov:query-keys-camel-case  rol__machtiging__loa is not lower camelCase.                                                   paths./zaken.get.parameters[46].name
  9043:15  error  nlgov:query-keys-camel-case  rol__omschrijvingGeneriek is not lower camelCase.                                              paths./zaken.get.parameters[47].name
  9065:15  error  nlgov:query-keys-camel-case  startdatum__gt is not lower camelCase.                                                         paths./zaken.get.parameters[49].name
  9071:15  error  nlgov:query-keys-camel-case  startdatum__gte is not lower camelCase.                                                        paths./zaken.get.parameters[50].name
  9077:15  error  nlgov:query-keys-camel-case  startdatum__lt is not lower camelCase.                                                         paths./zaken.get.parameters[51].name
  9083:15  error  nlgov:query-keys-camel-case  startdatum__lte is not lower camelCase.                                                        paths./zaken.get.parameters[52].name
  9089:15  error  nlgov:query-keys-camel-case  status__statustype is not lower camelCase.                                                     paths./zaken.get.parameters[53].name
  9102:15  error  nlgov:query-keys-camel-case  uiterlijkeEinddatumAfdoening__gt is not lower camelCase.                                       paths./zaken.get.parameters[55].name
  9109:15  error  nlgov:query-keys-camel-case  uiterlijkeEinddatumAfdoening__lt is not lower camelCase.                                       paths./zaken.get.parameters[56].name
  9121:15  error  nlgov:query-keys-camel-case  zaaktype__omschrijving is not lower camelCase.                                                 paths./zaken.get.parameters[58].name
 12371:15  error  nlgov:query-keys-camel-case  archiefactiedatum__gt is not lower camelCase.                                                  paths./zaken/_zoek.post.parameters[4].name
 12380:15  error  nlgov:query-keys-camel-case  archiefactiedatum__isnull is not lower camelCase.                                              paths./zaken/_zoek.post.parameters[5].name
 12388:15  error  nlgov:query-keys-camel-case  archiefactiedatum__lt is not lower camelCase.                                                  paths./zaken/_zoek.post.parameters[6].name
 12408:15  error  nlgov:query-keys-camel-case  archiefnominatie__in is not lower camelCase.                                                   paths./zaken/_zoek.post.parameters[8].name
 12429:15  error  nlgov:query-keys-camel-case  archiefstatus__in is not lower camelCase.                                                      paths./zaken/_zoek.post.parameters[10].name
 12445:15  error  nlgov:query-keys-camel-case  bronorganisatie__in is not lower camelCase.                                                    paths./zaken/_zoek.post.parameters[12].name
 12467:15  error  nlgov:query-keys-camel-case  einddatumGepland__gt is not lower camelCase.                                                   paths./zaken/_zoek.post.parameters[15].name
 12474:15  error  nlgov:query-keys-camel-case  einddatumGepland__lt is not lower camelCase.                                                   paths./zaken/_zoek.post.parameters[16].name
 12481:15  error  nlgov:query-keys-camel-case  einddatum__gt is not lower camelCase.                                                          paths./zaken/_zoek.post.parameters[17].name
 12487:15  error  nlgov:query-keys-camel-case  einddatum__isnull is not lower camelCase.                                                      paths./zaken/_zoek.post.parameters[18].name
 12492:15  error  nlgov:query-keys-camel-case  einddatum__lt is not lower camelCase.                                                          paths./zaken/_zoek.post.parameters[19].name
 12546:15  error  nlgov:query-keys-camel-case  identificatie__icontains is not lower camelCase.                                               paths./zaken/_zoek.post.parameters[22].name
 12558:15  error  nlgov:query-keys-camel-case  kenmerk__bron is not lower camelCase.                                                          paths./zaken/_zoek.post.parameters[24].name
 12631:15  error  nlgov:query-keys-camel-case  registratiedatum__gt is not lower camelCase.                                                   paths./zaken/_zoek.post.parameters[31].name
 12639:15  error  nlgov:query-keys-camel-case  registratiedatum__lt is not lower camelCase.                                                   paths./zaken/_zoek.post.parameters[32].name
 12647:15  error  nlgov:query-keys-camel-case  resultaat__resultaattype is not lower camelCase.                                               paths./zaken/_zoek.post.parameters[33].name
 12653:15  error  nlgov:query-keys-camel-case  rol__betrokkene is not lower camelCase.                                                        paths./zaken/_zoek.post.parameters[34].name
 12658:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__medewerker__identificatie is not lower camelCase.                paths./zaken/_zoek.post.parameters[35].name
 12665:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__natuurlijkPersoon__anpIdentificatie is not lower camelCase.      paths./zaken/_zoek.post.parameters[36].name
 12671:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__natuurlijkPersoon__inpA_nummer is not lower camelCase.           paths./zaken/_zoek.post.parameters[37].name
 12676:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn is not lower camelCase.                paths./zaken/_zoek.post.parameters[38].name
 12682:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__annIdentificatie is not lower camelCase.  paths./zaken/_zoek.post.parameters[39].name
 12688:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__innNnpId is not lower camelCase.          paths./zaken/_zoek.post.parameters[40].name
 12694:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__kvkNummer is not lower camelCase.         paths./zaken/_zoek.post.parameters[41].name
 12700:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__vestigingsNummer is not lower camelCase.  paths./zaken/_zoek.post.parameters[42].name
 12705:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__organisatorischeEenheid__identificatie is not lower camelCase.   paths./zaken/_zoek.post.parameters[43].name
 12712:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__vestiging__kvkNummer is not lower camelCase.                     paths./zaken/_zoek.post.parameters[44].name
 12718:15  error  nlgov:query-keys-camel-case  rol__betrokkeneIdentificatie__vestiging__vestigingsNummer is not lower camelCase.              paths./zaken/_zoek.post.parameters[45].name
 12724:15  error  nlgov:query-keys-camel-case  rol__betrokkeneType is not lower camelCase.                                                    paths./zaken/_zoek.post.parameters[46].name
 12737:15  error  nlgov:query-keys-camel-case  rol__machtiging is not lower camelCase.                                                        paths./zaken/_zoek.post.parameters[47].name
 12752:15  error  nlgov:query-keys-camel-case  rol__machtiging__loa is not lower camelCase.                                                   paths./zaken/_zoek.post.parameters[48].name
 12777:15  error  nlgov:query-keys-camel-case  rol__omschrijvingGeneriek is not lower camelCase.                                              paths./zaken/_zoek.post.parameters[49].name
 12799:15  error  nlgov:query-keys-camel-case  startdatum__gt is not lower camelCase.                                                         paths./zaken/_zoek.post.parameters[51].name
 12805:15  error  nlgov:query-keys-camel-case  startdatum__gte is not lower camelCase.                                                        paths./zaken/_zoek.post.parameters[52].name
 12811:15  error  nlgov:query-keys-camel-case  startdatum__lt is not lower camelCase.                                                         paths./zaken/_zoek.post.parameters[53].name
 12817:15  error  nlgov:query-keys-camel-case  startdatum__lte is not lower camelCase.                                                        paths./zaken/_zoek.post.parameters[54].name
 12823:15  error  nlgov:query-keys-camel-case  status__statustype is not lower camelCase.                                                     paths./zaken/_zoek.post.parameters[55].name
 12836:15  error  nlgov:query-keys-camel-case  uiterlijkeEinddatumAfdoening__gt is not lower camelCase.                                       paths./zaken/_zoek.post.parameters[57].name
 12843:15  error  nlgov:query-keys-camel-case  uiterlijkeEinddatumAfdoening__lt is not lower camelCase.                                       paths./zaken/_zoek.post.parameters[58].name
 12855:15  error  nlgov:query-keys-camel-case  zaaktype__omschrijving is not lower camelCase.                                                 paths./zaken/_zoek.post.parameters[60].name
```

--- a/.github/workflows/oas.yml
+++ b/.github/workflows/oas.yml
@@ -42,4 +42,4 @@ jobs:
       openapi-to-postman-version: "^5.0.0"
       postman-artifact-name: ${{ matrix.component }}-api-postman-collection
       openapi-generator-version: "^2.20.0"
-      spectral-ruleset: https://static.developer.overheid.nl/adr/2.1/ruleset.yaml
+      spectral-ruleset: ruleset.yaml

--- a/ruleset.yaml
+++ b/ruleset.yaml
@@ -1,0 +1,6 @@
+# 2.1 ruleset got changed without a version increase
+extends: https://static.developer.overheid.nl/adr/2.1/ruleset.yaml
+rules:
+  nlgov:query-keys-camel-case: warn
+  nlgov:paths-kebab-case: warn
+

